src/org/omegat/Bundle.properties

```diff
@@ -191,9 +191,6 @@ BUTTON_FILTER=&Filter
 BUTTON_REPLACE=&Show Replacements
 BUTTON_REPLACE_ALL=Re&place All
 
-BUTTON_SELECT=&OK
-BUTTON_SELECT_NO_MNEMONIC=Select
```

Comment:
these two is observed in ProjectPropertiesDialog.java and SearchWIndowContoller.java as commented out form.